website/server/models/task.js

```diff
@@ -58,6 +58,7 @@ export let TaskSchema = new Schema({
   userId: {type: String, ref: 'User', validate: [validator.isUUID, 'Invalid uuid.']}, // When not set it belongs to a challenge
 
   challenge: {
+    name: {type: String, default: 'None'},
```

Comment:
Could this be `shortName` so that it's really clear to future programmers? It would also need changing in website/server/models/challenge.js above.

One benefit would be that when you were grepping the source code for all places where shortName is used, this would show up.